app/src/main/java/org/schabi/newpipe/RouterActivity.java

```diff
@@ -572,7 +568,7 @@ private String trim(final String input) {
                 }
             }
         }
-        return result.toArray(new String[result.size()]);
+        return result.toArray(new String[0]);
```

Comment:
Why was this changed?

Reply:
![image](https://user-images.githubusercontent.com/32376686/91487190-5f8e3c80-e87b-11ea-8658-49bb02d561ca.png)

Reply:
Oh, OK